rosbag2_transport/src/rosbag2_transport/config_options_from_node_params.cpp

```diff
@@ -285,6 +287,15 @@ RecordOptions get_record_options_from_node_params(rclcpp::Node & node)
   }
   record_options.services = service_list;
 
+  // Covert action name to related topic name
+  auto action_list = node.declare_parameter<std::vector<std::string>>(
+    "record.actions", std::vector<std::string>());
+  for (auto & action : action_list) {
+    auto one_action_topic_list = rosbag2_cpp::action_name_to_action_topic_name(action);
+    std::move(one_action_topic_list.begin(), one_action_topic_list.end(),
+              std::back_inserter(record_options.actions));
+  }
```

Comment:
this is almost copied from service one above. does this do anything effective here? `action_list` is `std::vector<std::string>()`,and then `rosbag2_cpp::action_name_to_action_topic_name(action)` just returns `action` as it is. i may be mistaken but not really sure if this for loop is required to be here.

Reply:
rosbag2_cpp::action_name_to_action_topic_name(action) return action interface name (The name of this function has been modified to be clearer.）

For original code, record_options.actions want to save all action interface name. User set action name in record_options.actions by parameter.  So there is a conversion operation here to make record_options.actions include action interface name.  
Now, according to Micheal's suggestion (https://github.com/ros2/rosbag2/pull/1939#pullrequestreview-2708432340), I change it. record_options.actions include action name.  
But I found https://github.com/ros2/rosbag2/commit/4946f5fdb0ad772ed4a70af99c00e8f307f5aec3 missed fixing here. I will change codes.